spec/controllers/application_controller_spec.rb

```diff
@@ -486,7 +489,6 @@
 
     it 'fails if the feature is not found' do
       allow(controller).to receive(:features).and_return(features)
-      allow(controller).to receive(:assert_privileges).and_return(false)
```

Comment:
`assert_privileges` does the raising, how did this ever work?  The mocked method isn't needed.